terraform/etl/53-aws-glue-housing-extract-mmh-vulnerabilities-to-refined.tf

```diff
@@ -0,0 +1,67 @@
+locals {
+  housing_mtfh_case_notes_enriched_to_refined_environment_count = local.is_live_environment && !local.is_production_environment ? 1 : 0
+}
+
+module "housing_mtfh_case_notes_enriched_to_refined" {
+  count                          = local.housing_mtfh_case_notes_enriched_to_refined_environment_count
+  source                         = "../modules/aws-glue-job"
+  is_production_environment      = local.is_production_environment
+  is_live_environment            = local.is_live_environment
+  department                     = module.department_housing_data_source
+  job_name                       = "${local.short_identifier_prefix}Housing MTFH case notes enriched to refined"
+  script_s3_object_key           = aws_s3_object.housing_mtfh_case_notes_enriched_to_refined.key
+  glue_scripts_bucket_id         = module.glue_scripts_data_source.bucket_id
+  glue_temp_bucket_id            = module.glue_temp_storage_data_source.bucket_id
+  glue_job_timeout               = 360
+  helper_module_key              = data.aws_s3_object.helpers.key
+  pydeequ_zip_key                = data.aws_s3_object.pydeequ.key
+  spark_ui_output_storage_id     = module.spark_ui_output_storage_data_source.bucket_id
+  number_of_workers_for_glue_job = 2
+  glue_job_worker_type           = "G.1X"
+  glue_version                   = "3.0"
+  job_parameters                 = {
+    "--job-bookmark-option"              = "job-bookmark-enable"
+    "--enable-glue-datacatalog"          = "true"
+    "--enable-continuous-cloudwatch-log" = "true"
+    "--additional-python-modules"        = "great_expectations==0.15.48,spacy==3.7.4"
```

Comment:
Are you using **great_expectations** for data quality check or data profiling? Sorry, it's not a review comment. I want to learn from you about its use case. I may use it somewhere, as well. My former colleague did a short talk about it, but I don't have time to dive in. Thank you.

Reply:
I used a commercial app called [Anomalo](https://www.anomalo.com/) to do data quality check, you can find my former employer logo (Fandom) on its main page.
We planned to use [openmetadata ](https://open-metadata.org/)to do data quality check, but back then it was not meet our expectation so we went for commercial one.